std/functional.d

```diff
@@ -1376,6 +1379,33 @@ template memoize(alias fun, uint maxSize)
     assert(executed == 1);
 }
 
+// https://issues.dlang.org/show_bug.cgi?id=20302
+@system unittest
+{
+    version (unsupported) // TODO
```

Comment:
`none` ?

Reply:
It's effectively the same thing, but ok.